.github/CHANGELOG.md

```diff
@@ -182,32 +400,7 @@
    [ 0.69916862  0.34072424  0.69202359]]
   ```
 
-* A new quantum function transform has been added to push commuting
-  single-qubit gates through controlled operations.
-  [(#1464)](https://github.com/PennyLaneAI/pennylane/pull/1464)
-
-  The `commute_controlled` transform works as follows:
-
-  ```python
-  def circuit(theta):
-      qml.PauliX(wires=2)
-      qml.S(wires=0)
-      qml.CNOT(wires=[0, 1])
-      qml.PhaseShift(theta/2, wires=0)
-      qml.T(wires=0)
-      qml.Toffoli(wires=[0, 1, 2])
-      return qml.expval(qml.PauliZ(0))
-  ```
-
-  ```pycon
-  >>> optimized_circuit = qml.transforms.commute_controlled(direction="right")(circuit)
-  >>> dev = qml.device('default.qubit', wires=3)
-  >>> qnode = qml.QNode(optimized_circuit, dev)
-  >>> print(qml.draw(qnode)(0.5))
-   0: ──╭C──╭C──S──Rϕ(0.25)──T──┤ ⟨Z⟩
-   1: ──╰X──├C──────────────────┤
-   2: ──────╰X──X───────────────┤
-  ```
+<h4>Extended operations and templates</h4>
```

Comment:
```suggestion
<h4>Even more new operations and templates</h4>
```

Reply:
> A decomposition has been added to QubitUnitary that makes the single-qubit case fully differentiable in all interfaces. 

This one is really cool! Should we move it higher?

Reply:
Would you suggest moving it higher within the "Even more new operations and templates" section of outside of it even?

Reply:
I was thinking more moving it higher inside the "new operations" section :)

This one is something a lot of users have asked about (training qubit unitary), so is a nice one to make more visible